### PR TITLE
attachDom: added attach method to attach inputs of various types all at once

### PR DESCRIPTION
**Pull request by jrefano**

### CODE REVIEW

File: test/specs/trait/attachDom.js [hunk 1]
```diff
@@ -209,5 +209,35 @@ define([
         model.set('foo', 'baz');
       });
     });
+
+    ddescribe('attach', function() {
```
Comment (mrjoelkemp) [inline test/specs/trait/attachDom.js]: whoopsie

**Comment by mrjoelkemp** [comment]
Looks good by me. @Aintaer, the creator, would have a more careful eye.

File: trait/attachDom.js [hunk 2]
```diff
@@ -19,6 +19,14 @@ define(['jquery'], function($) {
 
   // Two-way binding of inputs to a model
   return {
+    attach: function($els) {
+      $els.each(function(i, el) {
+        if (el.nodeName.toLowerCase() === 'input') { this.attachText(el.name, $(el)); }
```
Comment (Aintaer) [inline trait/attachDom.js]: These should be testing against more specific selectors.
```
if ($el.is('input[type=text]')) {
  this.attachText(el.name, $el);
}
```

File: trait/attachDom.js [hunk 3]
```diff
@@ -19,6 +19,15 @@ define(['jquery'], function($) {
 
   // Two-way binding of inputs to a model
   return {
+    attach: function($els) {
+      $els.each(function(i, el) {
+        if (el.nodeName.toLowerCase() === 'input' && $(el).attr('type') === 'text') { this.attachText(el.name, $(el)); }
+        if (el.nodeName.toLowerCase() === 'input' && $(el).attr('type') === 'radio') { this.attachRadio(el.name, $(el)); }
```
Comment (Aintaer) [inline trait/attachDom.js]: This is not the way to go, it is non-maintainable.
```js
$els.each(function(i, el) {
  var $el = $(el);
  if ($el.is('input[type=text]')) {
    this.attachText(el.name, $el);
  }
  else if ($el.is('input[type=radio]')) {
    this.attachRadio(el.name, $el);
  }
  ...
```

Comment (mikesherov) [inline trait/attachDom.js]: Yeah, this is better. My version was quick and dirty slack chat.Using a map would be even more maintainbale

**Pull request merged**